indexd_ffi/src/lib.rs

```diff
@@ -988,10 +988,12 @@ impl SDK {
 
         let objects = objects
             .into_iter()
-            .map(|o| {
-                o.open(&self.app_key).map(|obj| {
-                    Arc::new(PinnedObject {
-                        inner: Arc::new(Mutex::new(obj)),
+            .filter_map(|event| {
```

Comment:
This method should be updated to events

Reply:
Adding a new event type is fine. I don't think he grabs the list of objects from the indexer yet.